machine_learning_hep/analysis/analyzerdhadrons_mult.py

```diff
@@ -649,8 +649,8 @@ def plot_tpc_tof_me(tag):
                 if tpc_tof_me:
                     for i in ["Pi", "K", "Pr"]:
                         for j in ["0", "1"]:
-                            for k in ["p", "pt"]:
-                                hname = [f"{k}_prong{j}",
+                            for k in ["p_prong0", "pt_prong0", "pt_cand"]:
+                                hname = [f"{k}",
```

Comment:
Hej @njacazio 
I was just wondering if you want to use python's `itertools.product` to only have one loop here? See e.g. https://stackoverflow.com/questions/798854/all-combinations-of-a-list-of-lists

Reply:
Hi, yes it can be done, maybe it helps in readability, I'm testing the commit now and will update the PR soon